logic/src/commonJvmAndroid/kotlin/com/wire/kalium/logic/CoreCryptoExceptionMapper.kt

```diff
@@ -32,6 +32,7 @@ actual fun mapMLSException(exception: Exception): MLSFailure =
             is CryptoError.StaleCommit -> MLSFailure.StaleCommit
             is CryptoError.ConversationAlreadyExists -> MLSFailure.ConversationAlreadyExists
             is CryptoError.MessageEpochTooOld -> MLSFailure.MessageEpochTooOld
+            is CryptoError.MlsException -> MLSFailure.InternalErrors
```

Comment:
question: should we do this to all internal errors, or only when it is related to sub conversation messages

Reply:
I think so, we can make it for this specific case only, but it's error prone as we will base on string